sdk/storage/azure-storage-blob/tests/test_largest_block_blob.py

```diff
@@ -259,7 +259,7 @@ def test_create_largest_blob_from_stream_single_upload_without_network(self, res
         payload_dropping_policy = PayloadDroppingPolicy()
         credential_policy = _format_shared_key_credential(storage_account.name, storage_account_key)
         self._setup(storage_account, storage_account_key, [payload_dropping_policy, credential_policy],
-                    max_single_put_size=LARGEST_SINGLE_UPLOAD_SIZE)
+                    max_single_put_size=LARGEST_SINGLE_UPLOAD_SIZE+1)
```

Comment:
why?

Reply:
I added this in an attempt to resolve the flaky test `test_create_largest_blob_from_stream_single_upload_without_network` which seems to sometimes do a `PUT` twice rather than once causing one of the assertions to fail.